frontend/src/diagram/sprotty/utils/geometry.tsx

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { Bounds, Point } from 'sprotty';
+
+export enum RectangleSide {
```

Comment:
Could you keep the types in a dedicated file named geometry.types.ts

Reply:
done